### DISCUSSION TITLE
Cannot link Render App to Github Repos

### DISCUSSION THREAD

**Discussion by BAK2K3** [community/community#39665] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I'm struggling to link Render to my Github account to authorise the repositories for deployment.

I've set up the account on Render, done the initial link to Github for the Render App, but when I go to select a repository (+ Connect an Account):

![image](https://user-images.githubusercontent.com/55312801/203072592-45aa5415-1f9a-4898-81c9-513de7121602.png)

I get the ol' Unicorn:

![image](https://user-images.githubusercontent.com/55312801/203072666-07b1ad00-1956-4fe1-a8d4-7de5ae24cc46.png)

The URL I'm trying to visit specifically is: https://github.com/apps/render/installations/new

I've tried:

- Removing the initial authorisation for the Render App and re-adding it
- Clearing my cache
- Edge, Chrome, Firefox
- Trying in Incognito/Private Mode
- I've checked the services of both Render and Github, and everything seems to be up and running
- I've checked with colleagues who are able to access the page. 

Any thoughts on what could be doing this, as it seems like it's something to do with my account that's blocking access to the app?

**Comment by subidit** [comment]
It might be that since github doesn't allow passwords to connect from external sources- I'd suggest that you might try with **Fine-grained personal access tokens**.

Try generating a token for a repo and then try to connect. Link - [Personal access tokens](https://github.com/settings/tokens?type=beta)

**Reply by BAK2K3** [reply]
Hi, thanks for responding.

I tried generating a Fine Grained token, but I don't seem to be able to use that token to generate the link between Render and Github.

On Render, you click a button to say "Connect account", and the initial connection is made automatically just by being logged in. You can see the initial connection in the authorised apps section of github with the following permissions:

- Verify your GitHub identity
- Know what resources you can access
- Act on your behalf
- View your email addresses



But it's as if it's not giving Render enough permissions. I have colleagues who don't seem to be having this issue, and aren't using the PATs as suggested. From the looks of things, I can authorise Render, but I just can't install it on any of my repos.

**Reply by BAK2K3** [reply]
![image](https://user-images.githubusercontent.com/55312801/203084405-1030b420-b8b1-442a-bf78-d6749365ca6e.png)

**Comment by BAK2K3** [comment, marked as answer]
I've fixed this issue. It was because I was part of CS50's me50 Organisation:

https://github.com/orgs/community/discussions/34751#discussioncomment-3797859

**Reply by subidit** [reply]
glad to meet another CS50 fellow. Which course did you take (also year)? Although I didn't finish the course but hope to get back on it someday. 
Cheers.

**Reply by BAK2K3** [reply]
2016, it was my intro to programming!
